plugins/catalog-backend-module-incremental-ingestion/src/module/WrapperProviders.ts

```diff
@@ -119,12 +119,17 @@ export class WrapperProviders {
         connection,
       });
 
-      const frequency = Duration.isDuration(burstInterval)
+      let frequency = Duration.isDuration(burstInterval)
         ? burstInterval
         : Duration.fromObject(burstInterval);
-      const length = Duration.isDuration(burstLength)
+      if (frequency.as('milliseconds') < 5000) {
```

Comment:
Is there a risk that 5 is a bit high?

Reply:
So what happens is that it'll take up to five seconds longer than you thought between bursts - that is, the interval that's meant to be an actual rest time between intense many-page pagination runs. I felt like resting for less than five seconds, e.g. to stave off rate limits or not exhaust resources, kinda wouldn't achieve much most likely 🤷 I hope that this duration most of the time is set to like on the order of a minute or several minutes, but not sure

Reply:
Let's merge and experiment I guess, happy to lower it later on if needed